### DISCUSSION TITLE
layer control: default display status

### DISCUSSION THREAD

**Discussion by shi093** [opengeos/leafmap#80] [Q&A] [answered]
When I load a layer to the map using add_geojson(), is there a way to specify whether the layer is turned on/off when the map is rendered?

**Comment by giswqs** [comment, marked as answer]
Unfortunately, the answer is NO. You can only turn on/off XYZ/WMS layers programmatically. GeoJSON/GeoData layers do not support this. See https://github.com/jupyter-widgets/ipyleaflet/issues/827
